features/fork.feature

```diff
@@ -202,3 +202,22 @@ Scenario: Related fork already exists
     And "git.my.org" is a whitelisted Enterprise host
     When I successfully run `hub fork`
     Then the url for "mislav" should be "git@git.my.org:mislav/dotfiles.git"
+
+  Scenario: --org github
```

Comment:
ok, I was following the no-remote scenario style... But I prefer the one you propose